src/sources/forus-platform/pug/tpl/directives/website/footer.pug

```diff
@@ -1,252 +1,48 @@
-                                span.help-block
-                    .col-xs-12
-                        .form-group
-                            label.control-label ORGANISATIE
-                            .input-group
-                                input#companyInput.form-control(placeholder="Organisatie" name="organization" value="" onkeyup="this.setAttribute('value', this.value);" )
-                                a(href="#").clear_field
-                                    include ../../../../resources/platform-general/assets/img/svg/clear-field.svg
-                                span.help-block
-                    .col-xs-12
-                        .form-group
-                            label.control-label TELEFOONNUMMER
-                            .input-group
-                                input#telInput.form-control(placeholder="Telefoonnummer" name="phone" value="" onkeyup="this.setAttribute('value', this.value);" )
-                                a(href="#").clear_field
-                                    include ../../../../resources/platform-general/assets/img/svg/clear-field.svg
-                                span.help-block
-                    .col-xs-12
-                        .form-group.required
-                            label.control-label BERICHT
-                            .input-group
-                                textarea#messageInput.form-control(placeholder="Bericht" name="message")
-
-                                a(href="#").clear_field
-                                    include ../../../../resources/platform-general/assets/img/svg/clear-field.svg
-                                span.help-block
-                    .col-xs-12.info
-                        span(style="color: red") *
-                        | velden zijn verplicht
-                    .col-xs-12
-                        .form-group.center-block
-                            p.thanks Bedankt! Er wordt zo snel mogelijk contact met u opgenomen.
-                            button#formSubmitHome.main_button.g-recaptcha( data-sitekey="6LfA-JcUAAAAAC9SpgAr2ZmL4rFwjb83_U2MDcfQ" data-callback="onSubmitCFHome" data-badge="inline" type="button") Versturen
-footer(class="footer_sub")
+.mr-tp
+  .footer
     .container
-        nav.navbar
-            .container-fluid
-                .navbar-header
-                    a.navbar-brand(href="/"): include ../../../../resources/platform-general/assets/img/svg/forus-logo-footer.svg
-                #navbar_footer.navbar-collapse
-                    ul.nav.navbar-nav
-                        //li
-                            a(href="#") Privacy & Terms
-
-                        //li
-                            a(href="#") Contact Us
-
+      .col-6.col-md-4
+        h5 Contactgegevens
+        ul
+          li: p E-mailadres: <span><a href="mailto:info@forus.io">info@forus.io</a></span>
```

Comment:
does this not work?
```pug
li: p 
    | E-mailadres: 
    span
       a(href="mailto:info@forus.io")
       | support@forus.io
```

Reply:
Well check it out locally if you want to know